third_party/move/move-model/bytecode/src/stackless_bytecode.rs

```diff
@@ -475,15 +475,13 @@ impl Bytecode {
             Bytecode::Ret(..)
                 | Bytecode::Jump(..)
                 | Bytecode::Abort(..)
+                | Bytecode::Branch(..)
```

Comment:
It is confusing (and incorrect, IMO) to call a `Branch` with two targets that depends on a condition an "unconditional branch".

Reply:
It is however true that `Branch` ends a basic block, as there is no fall-through control flow.  Perhaps `is_unconditional_branch` could be renamed to `always_redirects_control_flow` and and `is_conditional_branch` could be renamed to `may_redirect_control_flow`.

Reply:
Renamed